doc/source/missing_data.rst

```diff
@@ -170,52 +170,38 @@ The descriptive statistics and computational methods discussed in the
 account for missing data. For example:
 
 * When summing data, NA (missing) values will be treated as zero.
-* If the data are all NA, the result will be NA.
-* Methods like **cumsum** and **cumprod** ignore NA values, but preserve them
-  in the resulting arrays.
+* If the data are all NA, the result will be 0.
+* Cumulative methods like :meth:`~DataFrame.cumsum` and :meth:`~DataFrame.cumprod` ignore NA values by default, but preserve them in the resulting arrays. To override this behaviour and include NA values, use ``skipna=False``.
 
 .. ipython:: python
 
    df
    df['one'].sum()
    df.mean(1)
    df.cumsum()
+   df.cumsum(skipna=False)
 
 
 .. _missing_data.numeric_sum:
 
 Sum/Prod of Empties/Nans
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. warning::
-
-   This behavior is now standard as of v0.21.0; previously sum/prod would give different
-   results if the ``bottleneck`` package was installed.
-   See the :ref:`v0.21.0 whatsnew <whatsnew_0210.api_breaking.bottleneck>`.
-
-With ``sum`` or ``prod`` on an empty or all-``NaN`` ``Series``, or columns of a ``DataFrame``, the result will be all-``NaN``.
-
-.. ipython:: python
-
-   s = pd.Series([np.nan])
-
-   s.sum()
-
-Summing over an empty ``Series`` will return ``NaN``:
+With ``sum`` on an empty or all-``NaN`` ``Series``, or columns of a ``DataFrame``, the result will be 0.
 
 .. ipython:: python
 
+   pd.Series([np.nan]).sum()
+   
    pd.Series([]).sum()
 
-.. warning::
-
-   These behaviors differ from the default in ``numpy`` where an empty sum returns zero.
+With ``prod`` on an empty or all-``NaN`` ``Series``, or columns of a ``DataFrame``, the result will be 1.
```

Comment:
Similar

    The product of an empty or all-NA Series or column of a DataFrame is 1.